BaseClasses.py

```diff
@@ -1243,6 +1245,7 @@ def __str__(self) -> str:
 class Spoiler():
     multiworld: MultiWorld
     unreachables: Set[Location]
+    plando_settings: PlandoSettings
```

Comment:
I don't like having a member that is not initialized in `__init__`

I wonder why this info isn't attached to the instance of `MultiWorld`

Reply:
i did that at first but didn't like it... the only other alternative i have is passing it as an arg to the init which i also didn't like. i could just assign it in the class body i suppose? probably should just for safety